frontend/src/metabase/core/components/AccordionList/AccordionListCell.jsx

```diff
@@ -122,12 +122,15 @@ export const AccordionListCell = ({
             </span>
           )}
           {name && (
-            <h3
+            <Text
               data-element-id="list-section-title"
               className={cx(ListS.ListSectionTitle, CS.textWrap)}
+              fz="lg"
+              lh="normal"
```

Comment:
Is the line-height not set to `normal` by default?

Reply:
Nope 🙃 I still have no idea why we thought this was a good idea 

![image](https://github.com/user-attachments/assets/65888728-a8d7-48d4-ba85-9f1867b23205)